### PR TITLE
Adjust courseList filter

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes

Addresses comment from @pandasa123 
[shown here](https://github.com/Qiskit/platypus/pull/855#discussion_r855498899)

## Implementation details

- updates the filter to get everything _but_ docs

## How to read this PR

- review filter
- review UI (or screen capture)

## Screenshots

**BEFORE filter updates**

https://user-images.githubusercontent.com/6276074/164539019-34166b6c-64d0-4de8-87e6-c58149e7364d.mov

**AFTER filter updates (this PR)**

https://user-images.githubusercontent.com/6276074/164539142-2fb4d369-994b-4126-af84-f2cca693d558.mov

### CODE REVIEW

**Comment by pandasa123** [comment]
Looks great!!

**Review by pandasa123 — APPROVED** [review]

File: frontend/vue/components/Syllabus/SyllabusView.vue [hunk 1]
```diff
@@ -68,7 +68,7 @@ export default defineComponent({
   },
   mounted () {
     getCourseList().then((courses) => {
-      const learningPathCourses = courses.filter(course => course.type === 'learning-path')
+      const learningPathCourses = courses.filter(course => course.type !== 'docs')
```
Comment (vabarbosa) [inline frontend/vue/components/Syllabus/SyllabusView.vue:71]: maybe rename variable to `availableCourses` or something along those lines

**Review by vabarbosa — CHANGES_REQUESTED** [review]

File: frontend/vue/components/Syllabus/SyllabusView.vue (hunk 1 above)
Comment (techtolentino) [reply frontend/vue/components/Syllabus/SyllabusView.vue]: great suggestion @vabarbosa 👏

**Review by vabarbosa — APPROVED** [review]
@techtolentino thank you!

**Pull request merged**